src/chrome/content/rules/BostonGlobe.com.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+	Related:
```

Comment:
You can just make a reference to the list in Boston.com.xml. Otherwise we have to maintain the list in all rulesets.